src/main/java/world/bentobox/bentobox/lists/Flags.java

```diff
@@ -80,6 +80,7 @@ private Flags() {}
      * @see BreakBlocksListener
      */
     public static final Flag BREAK_BLOCKS = new Flag.Builder("BREAK_BLOCKS", Material.STONE_PICKAXE).listener(new BreakBlocksListener()).mode(Flag.Mode.BASIC).build();
+    public static final Flag BREAK_SPAWNERS = new Flag.Builder("BREAK_SPAWNERS", Material.SPAWNER).listener(new BreakBlocksListener()).mode(Flag.Mode.BASIC).build();
```

Comment:
```suggestion
    /**
     * Prevents players from breaking spawners on one's island.
     * @see BreakBlocksListener
     * @since 1.13.0
     */
    public static final Flag BREAK_SPAWNERS = new Flag.Builder("BREAK_SPAWNERS", Material.SPAWNER).mode(Flag.Mode.EXPERT).build();
```
Duplicating the listener here is not needed. Moreover, I don't really consider this flag to be a "basic" one. It's more of an advanced one, or even expert. Here, let's go with Expert. Plus, let's add some proper javadoc ;)